### DISCUSSION TITLE
v4.1.1

### DISCUSSION THREAD

**Discussion by epicfaace** [rjsf-team/react-jsonschema-form#2781] [Announcements]
## @rjsf/material-ui
- Fix bloated bundle size by individually requiring all MUI components (#2754)
- Add new `useMuiComponent()` hook as a shortcut for `useContext(MuiComponentContext)`

## New Contributors
* @rollercodester made their first contribution in https://github.com/rjsf-team/react-jsonschema-form/pull/2754
* @kokoyroy made their first contribution in https://github.com/rjsf-team/react-jsonschema-form/pull/2770

**Full Changelog**: https://github.com/rjsf-team/react-jsonschema-form/compare/v4.0.1...v4.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/rjsf-team/react-jsonschema-form/releases/tag/v4.1.1'>v4.1.1</a>.</em>

**Comment by szubster** [comment]
This version did not land on npmjs

**Reply by epicfaace** [reply]
For some reason, the github actions pipeline didn't trigger. Triggering it again by re-releasing: https://github.com/rjsf-team/react-jsonschema-form/actions/workflows/release.yml

**Reply by szubster** [reply]
Thank you!
